TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -349,5 +349,31 @@ public LockTableStatement CreateLockTableStatement(CodeElementsParser.LockTableS
 
             return new LockTableStatement(tableName, partitionId, mode);
         }
+
+        public DropTableStatement CreateDropTableStatement(CodeElementsParser.DropTableStatementContext context)
+        {
+            if (context.tableOrAliasName() == null) return null;
+            var tableOrAliasName = CreateTableOrAliasName(context.tableOrAliasName());
+            return new DropTableStatement(tableOrAliasName);
+        }
+
+        private TableViewCorrelationName CreateTableOrAliasName(CodeElementsParser.TableOrAliasNameContext context)
+        {
+            if (context.tableOrViewOrCorrelationName().Name == null) return null;
+            var nameToken = (Token)context.tableOrViewOrCorrelationName().Name;
+            if (context.tableOrViewOrCorrelationName().SchemaName != null)
+            {
+                var qualifierToken = (Token) context.tableOrViewOrCorrelationName().SchemaName;
+                var topLevelQualifierToken = (Token)context.tableOrViewOrCorrelationName().DBMS;
+                return new TableViewCorrelationName(CreateSymbolReference(nameToken,qualifierToken,topLevelQualifierToken));
+            }
+            else
```

Comment:
Since if block above always returns, the else block is redundant. Look for ReSharper warnings.